aspnetcore/grpc/browser/sample/AllServicesSupportExample_Startup.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using Server.Services;
+
+namespace Server
+{
+    #region snippet_1
+

```

Comment:
remove empty lines around snippets

Reply:
Removing empty lines on snippets, and will update the number highlighting in a moment to reflect the change.